src/lib/core/questions.ts

```diff
@@ -642,6 +642,19 @@ export const questionGroups: QuestionGroup[] = [
 							: null,
 					].filter((f) => !!f) as string[],
 			}),
+			{
+				type: "select",
+				name: "releaseScript",
+				label: "Release Script",
+				message:
+					"Would you like to create new releases directly from the command line?",
```

Comment:
Maybe "sell" this option a bit more? What do you think?
```suggestion
					"Would you like to automate new releases with one simple command?",
```